CPT331.WebAPI/CPT331.WebAPI.csproj

```diff
@@ -125,8 +143,27 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Content\bootstrap-theme.css" />
```

Comment:
I see what you now mean about getting the site 'working' - my definition was that it compiled, I didn't try and run it or look at any views. This PR makes a lot more sense now.